### PR TITLE
chore: prepare to publish component packages

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

**Review by dgozman — APPROVED** [review]

File: packages/playwright-ct-vue/package.json [hunk 1]
```diff
@@ -1,8 +1,22 @@
 {
-    "name": "@playwright/ct-vue",
+    "name": "@playwright/experimental-ct-vue",
```
Comment (mxschmitt) [inline packages/playwright-ct-vue/package.json:2]: The last time we prefixed a package with experimental it went wrong and people kept using it for years in blog articles (allure). Also they didn't known about it's successor.

Let's use the right name but a 0.x version which is more common?

File: tests-components/ct-react-vite/package.json [hunk 2]
```diff
@@ -17,5 +17,9 @@
     "@vitejs/plugin-react": "^1.0.7",
     "typescript": "^4.5.4",
     "vite": "^2.8.0"
+  },
+  "@standaloneDevDependencies": {
```
Comment (mxschmitt) [inline tests-components/ct-react-vite/package.json:21]: Who invented that?

File: packages/playwright-ct-vue/package.json (hunk 1 above)
Comment (pavelfeldman) [reply packages/playwright-ct-vue/package.json:2]: The problem is that we don't yet have the 'right name'.

File: tests-components/ct-react-vite/package.json (hunk 2 above)
Comment (pavelfeldman) [reply tests-components/ct-react-vite/package.json:21]: me!

**Pull request merged**